### PR TITLE
Add content return value to doco

### PR DESCRIPTION
**Pull request by beachrunner2k**
##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

`lib/ansible/plugins/action/uri.py`

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by acozine** [comment]
@foo2k thanks for your interest in improving the Ansible documentation. Lines 321-325 are not valid YAML because they are flush with the left margin - those lines would need to be properly indented for this PR to pass our test suite. I see you've used our "edit on GitHub" option - are you comfortable pulling your branch and updating it?

File: lib/ansible/modules/net_tools/basics/uri.py [hunk 1]
```diff
@@ -313,6 +313,16 @@
 
 RETURN = r'''
 # The return information includes all the HTTP headers in lower-case.
+content:
+  description: The content of the requested resource if return_content is true
+  returned: When return_content is TRUE. 
+  type: string
+  sample: <!DOCTYPE html>
```
Comment (acozine) [inline lib/ansible/modules/net_tools/basics/uri.py]:
```suggestion
  sample: 
     <!DOCTYPE html>
```

File: lib/ansible/modules/net_tools/basics/uri.py [hunk 2]
```diff
@@ -313,6 +313,16 @@
...
+<html>
```
Comment (acozine) [inline lib/ansible/modules/net_tools/basics/uri.py]:
```suggestion
     <html>
```

File: lib/ansible/modules/net_tools/basics/uri.py [hunk 3]
```diff
@@ -313,6 +313,16 @@
...
+<body>
```
Comment (acozine) [inline lib/ansible/modules/net_tools/basics/uri.py]:
```suggestion
     <body>
```

File: lib/ansible/modules/net_tools/basics/uri.py [hunk 4]
```diff
@@ -313,6 +313,16 @@
...
+Hello world.
```
Comment (acozine) [inline lib/ansible/modules/net_tools/basics/uri.py]:
```suggestion
     Hello world.
```

File: lib/ansible/modules/net_tools/basics/uri.py [hunk 5]
```diff
@@ -313,6 +313,16 @@
...
+</body>
```
Comment (acozine) [inline lib/ansible/modules/net_tools/basics/uri.py]:
```suggestion
     </body>
```

File: lib/ansible/modules/net_tools/basics/uri.py [hunk 6]
```diff
@@ -313,6 +313,16 @@
...
+</html>
```
Comment (acozine) [inline lib/ansible/modules/net_tools/basics/uri.py]:
```suggestion
     </html>
```

File: lib/ansible/modules/net_tools/basics/uri.py [hunk 7]
```diff
@@ -313,6 +313,16 @@
 
 RETURN = r'''
 # The return information includes all the HTTP headers in lower-case.
+content:
+  description: The content of the requested resource if return_content is true
+  returned: When return_content is TRUE.
```
Comment (acozine) [inline lib/ansible/modules/net_tools/basics/uri.py]:
```suggestion
  returned: When return_content is TRUE.
```

File: lib/ansible/modules/net_tools/basics/uri.py [hunk 8]
```diff
@@ -337,7 +347,7 @@
   description: The actual URL used for the request
   returned: always
   type: str
-  sample: https://www.ansible.com/
+  sample: https://www.ansible.com/
```
Comment (acozine) [inline lib/ansible/modules/net_tools/basics/uri.py]:
```suggestion
  sample: https://www.ansible.com/
```

**Review by acozine — COMMENTED** [review]
I think if you accept all these suggestions, your PR will pass our CI suite. You have to click "commit suggestion", not "resolve conversation" to accept each suggestion.

**Comment by samccann** [comment]
Hi @foo2k  - Sorry for the delay on this one. Looks like another PR addressed this - https://github.com/ansible/ansible/pull/66612 so closing this PR out. Thanks again for your help with Ansible!

**Comment by foo2k** [comment]
Sorry, but isn't [#66612](https://github.com/ansible/ansible/pull/66612) referring back to this PR?!

**Comment by samccann** [comment]
If you mean the last reference in that PR, that is because I mentioned it in this PR so github added the reference.

**Pull request closed without merging**